### PR TITLE
Refactor analysis for per turn eval

### PR DESCRIPTION
**Pull request by mus1cholic**
**Patch description**
Refactors code for processing left_pane_text so code injections can be made on other crowdsourcing tasks

**Testing steps**
See CircleCI tests

### CODE REVIEW

File: parlai/crowdsourcing/tasks/acute_eval/webapp/package-lock.json
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/acute_eval/webapp/package-lock.json]: This is a generated file - we don't want to include it in the PR

File: parlai/crowdsourcing/tasks/model_chat/conf/example.yaml
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/conf/example.yaml]: let's keep the example the same - we want the casual user of model chat to not have to load the large Blender model

File: parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py:254]: hmm, honestly, I think that these 3 lines of code will probably be the same for all subclasses of model_chat, meaning that the lines will need to get repeated in all subclasses. I think it'll lead to less duplicated code if, instead, there's a method called something like `format_left_pane_text()`, which you just define as `pass` for this superclass

File: parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml]: Good to not add in these commented-out lines =P

**Review by EricMichaelSmith — COMMENTED** [review]
Hi Orion! This is a great idea - but I think there's a slightly different setup that will lead to even less code-reuse :)

File: parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml
Comment (EricMichaelSmith) [inline parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml:3]: Nit: extra space

**Review by stephenroller — COMMENTED** [review]
I concur with Eric's comments

**Review by stephenroller — APPROVED** [review]

File: parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml
Comment (mus1cholic) [reply parlai/crowdsourcing/tasks/model_chat/task_config/model_opts.yaml]: Resolved

File: parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py
Comment (mus1cholic) [reply parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py]: Resolved

**Review by EricMichaelSmith — APPROVED** [review]
Looks good, thanks! But make sure to wait until all CI checks pass before merging in :)

**Comment by mus1cholic** [comment]
CI tests passed, merging in now

**Pull request merged**
